server/py/framework/db/sqldb/models.py

```diff
@@ -1068,6 +1067,35 @@ def _sqlite_autoincrement(
         target.id = next_id
 
 
+@event.listens_for(AlertActivation.__table__, "after_create")
+def bootstrap_partitions(table, connection, **_):
+    dialect = connection.dialect.name
+    if dialect not in (
+        mlrun.common.db.dialects.Dialects.MYSQL,
+        mlrun.common.db.dialects.Dialects.POSTGRESQL,
+    ):
+        return
```

Comment:
can be checked in a separate function. Also, do we want to say something if dialect is not in the list of the expected once?